File: Python/series_sum.py
# # SOLUTION ONE
# def series_sum(n):
#     a = 1.0
#     res = 0
#     i = 0
#     while i < n:
#         res += (1.0 / a)
#         a += 3
#         i += 1
#     return "%.2f" % round(res, 2)

# SOLUTION TWO
def series_sum(n):
    return '{:.2f}'.format(sum(1.0/(3 * i + 1) for i in range(n)))


print(series_sum(1))
print(series_sum(2))
print(series_sum(5))

File: JavaScript/seriesSum.js
function SeriesSum(n) {
  var d = 1.0;
  var s = 0;
  for (let i = 1; i <= n; i++) {
    s += 1 / d;
    d += 3.0;
  }
  return `${s.toFixed(2)}`;
}

console.log(SeriesSum(1));
console.log(SeriesSum(2));
console.log(SeriesSum(5));
